src/main/java/com/ufcg/es5/BackendComplexoEsportivoUFCG/exception/ErrorHandlingControllerAdvice.java

```diff
@@ -57,28 +66,28 @@ public CustomErrorType onResourceNotFoundException(NoHandlerFoundException e) {
         );
     }
 
-    @ExceptionHandler(ComplexoEspExceptionBadRequest.class)
+    @ExceptionHandler(SaceInvalidArgumentException.class)
     @ResponseStatus(HttpStatus.BAD_REQUEST)
     @ResponseBody
-    public CustomErrorType onEspExceptionBadRequest(ComplexoEspExceptionBadRequest e) {
+    public CustomErrorType onBadRequestException(SaceInvalidArgumentException e) {
         return defaultCustomErrorTypeConstruct(
-               "Bad request" + e.getMessage()
+                "Bad request" + e.getMessage()
         );
     }
 
-    @ExceptionHandler(ComplexoEspExceptionNotFound.class)
+    @ExceptionHandler(SaceResourceNotFoundException.class)
     @ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
Not found retorna 404, que significa que o endpoint que esta sendo acessado não foi encontrado, acredito que esse erro será usado quando buscarmos algo no backend que não exista, então deve retornar 400, Bad request.